bin/critterget.py

```diff
@@ -38,23 +41,23 @@ def apicall (uri, attribs=''):
 
     if (debug) : print u'reqstring is {}'.format(reqstring)
 
-    request = urllib2.Request(reqstring)
-    request.add_header('Content-Type','application/json')
-    request.add_header('Authorization', "Bearer %s" % access_token)
-    request.add_header('CR-source', 'integration_splunk')
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': "Bearer %s" % access_token,
+        'CR-source': 'integration_splunk'
+    }
 
     try:
-        response = urllib2.urlopen(request)
-        resptext = response.read()
-        data = json.loads(resptext)
+        response = requests.get(reqstring, headers=headers)
+        data = response.json()
 
-    except urllib2.URLError, e:
+    except requests.exceptions.RequestException as e:
         print 'Crittercism API retuned an error code:', e
         sys.exit(0)
 
     return data
 
-def apipost (uri, postdata='',keyget=''):
+def apipost (uri, postdata='', keyget=''):
```

Comment:
Whenever you're modifying a function like this clean it up a little if you can.

I would replace the default values for postdata and keyget with None here.